vm_automation/esxiVm.py

```diff
@@ -465,11 +465,11 @@ def revertToTestingBase(self):
     def revertToSnapshot(self, snapshotObj):
         return self.waitForTask(snapshotObj.RevertToSnapshot_Task())
 
-    def revertToSnapshotByName(self, snapshotObj):
+    def revertToSnapshotByName(self, snapshotName):
         self.server.logMsg("RESETTING VM " + self.vmName)
         self.getSnapshots()
         for snapshotObject in self.snapshotList:
-            if 'testing_base' in snapshotObject[0].name.lower():
+            if snapshotName.lower() in snapshotObject[0].name.lower():

```

Comment:
This seems possibly ambiguous consider:
```
VM --> | --> OldSnapshot1
             | --> Snapshot1
             | --> Snapshot15
 ```

```
vmserver.revertToSnapshotByName("Snapshot1")
```

I believe this would result in restore to `OldSnapshot1`

I don't mind not being case sensitive, however we should require a full string match here.

Reply:
Ewwwww. Yeah; not sure what I was thinking, there.... Thank you for the catch.